docs/tasks.md

```diff
@@ -68,6 +68,7 @@ The following tables gives you an overview of the tasks in MTEB.
 | [CmedqaRetrieval](https://aclanthology.org/2022.emnlp-main.357.pdf) | ['cmn'] | Retrieval | s2p |  |  |  |
 | [Cmnli](https://huggingface.co/datasets/clue/viewer/cmnli) | ['cmn'] | PairClassification | s2s |  |  |  |
 | [CodeSearchNetRetrieval](https://huggingface.co/datasets/code_search_net/viewer) (Husain et al., 2019) | ['go', 'java', 'javascript', 'php', 'python', 'ruby'] | Retrieval | p2p | [Programming] | {'test': 1000} | {'test': 1196.4609} |
+| [CodeEditSearchRetrieval](https://huggingface.co/datasets/cassanof/CodeEditSearch/viewer) (Muennighoff el al., 2023) | ['python', 'javascript', 'typescript', 'go', 'ruby', 'java', 'php', 'c', 'c++', 'rust', 'swift', 'scala', 'shell'] | Retrieval | p2p | [Programming] | {'train': 1000} | {'train': 553.50} |
```

Comment:
This table is autogenerated - so this change will be autogenerated

Reply:
Ok! I will delete